brigade-worker/node_modules/@brigadecore/brigadier/out/events.d.ts

```diff
@@ -129,6 +129,8 @@ export interface Repository {
      * sshKey the SSH key to use for ssh:// or git+ssh:// protocols
      */
     sshKey?: string;
+
+    sshCert?: string;
```

Comment:
This file was amended to get tests passing, right?  Let's remember to remove this change (perhaps after bumping the brigadier version if https://github.com/brigadecore/brigadier/pull/33 is merged first).

Reply:
correct, or we can add temporary branch to brigadier and use it in  yarn, but thats probably an overkill...

Reply:
Ok, now that https://github.com/brigadecore/brigadier/pull/33 is merged, let me get an npm release out there and then we can update this branch and we should be looking good... will report back once release is cut.

Reply:
cool

Reply:
Ok @vmasutin - https://www.npmjs.com/package/@brigadecore/brigadier/v/0.9.0 is out.  Please update the dep in this branch and rm the manual override.  Let's run CI once that is done.  Thanks!

Reply:
I have pushed the change